### ISSUE TITLE
Could not import the Aer provider from the qiskit-aer package 

### ISSUE THREAD

**Issue by zohimchandani** [Qiskit/qiskit-aer#1797]
### Informations

- **Qiskit Aer version**: 0.12.0
- **Python version**: 3.10.6 
- **Operating system**: Ubuntu 22.04.2

### What is the current behavior?

Trying to run the code in [this](https://learn.qiskit.org/course/ch-applications/hybrid-quantum-classical-neural-networks-with-pytorch-and-qiskit) notebook however I am getting the following error when installing the packages with specific versions as required by the notebook. 

### Steps to reproduce the problem

The notebook linked requires certain versions which I install via 

```
pip install qiskit==0.27.0 qiskit-terra==0.17.4 qiskit-aer==0.8.2 qiskit-ignis==0.6.0 qiskit-aqua==0.9.2 qiskit-ibmq-provider==0.14.0

Requirement already satisfied: python-dateutil>=2.8.0 in /home/cudaq/.local/lib/python3.10/site-packages (from qiskit-terra==0.17.4) (2.8.2)
...
  ERROR: Failed building wheel for qiskit-aer
  Running setup.py clean for qiskit-aer
Failed to build qiskit-aer
ERROR: Could not build wheels for qiskit-aer, which is required to install pyproject.toml-based projects
```

When I run the code from the notebook, I get the following error: 

```
simulator = qiskit.Aer.get_backend('aer_simulator')

circuit = QuantumCircuit(1, simulator, 100)
print('Expected value for rotation pi {}'.format(circuit.run([np.pi])[0]))
circuit._circuit.draw()
```

```
/tmp/ipykernel_104414/1061457973.py:30: DeprecationWarning: Using a qobj for run() is deprecated as of qiskit-aer 0.9.0 and will be removed no sooner than 3 months from that release date. Transpiled circuits should now be passed directly using `backend.run(circuits, **run_options).
  job = self.backend.run(qobj)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[9], line 4
      1 simulator = qiskit.Aer.get_backend('aer_simulator')
      3 circuit = QuantumCircuit(1, simulator, 100)
----> 4 print('Expected value for rotation pi {}'.format(circuit.run([np.pi])[0]))
      5 circuit._circuit.draw()

Cell In[7], line 30, in QuantumCircuit.run(self, thetas)
     25 t_qc = transpile(self._circuit,
     26                  self.backend)
     27 qobj = assemble(t_qc,
     28                 shots=self.shots,
     29                 parameter_binds = [{self.theta: theta} for theta in thetas])
---> 30 job = self.backend.run(qobj)
     31 result = job.result().get_counts()
     33 counts = np.array(list(result.values()))

File ~/.local/lib/python3.10/site-packages/qiskit_aer/backends/aerbackend.py:196, in AerBackend.run(self, circuits, validate, parameter_binds, **run_options)
    194         if 'parameter_binds' in run_options:
    195             parameter_binds = run_options.pop('parameter_binds')
--> 196     return self._run_qobj(circuits, validate, parameter_binds, **run_options)
    198 only_circuits = True
    199 only_pulse = True

TypeError: AerBackend._run_qobj() got multiple values for argument 'parameter_binds'
```

### What is the expected behavior?

Can you give me an option to download the notebook and run it as expected please. 

### Suggested solutions

NA

**Comment by hhorii** [comment]
> TypeError: AerBackend._run_qobj() got multiple values for argument 'parameter_binds'
This error will be resolved when https://github.com/Qiskit/qiskit-aer/pull/1792 is merged.

**Comment by hhorii** [comment]
```
class QuantumCircuit:
    """ 
    This class provides a simple interface for interaction 
    with the quantum circuit 
    """
    
    def __init__(self, n_qubits, backend, shots):
        # --- Circuit definition ---
        self._circuit = qiskit.QuantumCircuit(n_qubits)
        
        all_qubits = [i for i in range(n_qubits)]
        self.theta = qiskit.circuit.Parameter('theta')
        
        self._circuit.h(all_qubits)
        self._circuit.barrier()
        self._circuit.ry(self.theta, all_qubits)
        
        self._circuit.measure_all()
        # ---------------------------

        self.backend = backend
        self.shots = shots
    
    def run(self, thetas):
        t_qc = transpile(self._circuit,
                         self.backend)
        qobj = assemble(t_qc,
                        shots=self.shots,
                        parameter_binds = [{self.theta: theta} for theta in thetas])
        job = self.backend.run(qobj)
        result = job.result().get_counts()
        
        counts = np.array(list(result.values()))
        states = np.array(list(result.keys())).astype(float)
        
        # Compute probabilities for each state
        probabilities = counts / self.shots
        # Get state expectation
        expectation = np.sum(states * probabilities)
        
        return np.array([expectation])
```

The above code in the tutorial uses qobj. I think we can run circuits directly as follows:

```
        t_qc = transpile(self._circuit,
                         self.backend)
        job = self.backend.run(t_qc,
                        shots=self.shots,
                        parameter_binds = [{self.theta: theta} for theta in thetas])
```

**Comment by hhorii** [comment]
#1792 was released in 0.12.1.
